content/applications/essentials/contacts.rst

```diff
@@ -0,0 +1,219 @@
+
+Membership tab
+--------------
+
+Finally, is the :guilabel:`Membership` tab, which can help to manage any memberships that are being
+offered to this specific contact. It should be noted that this tab is associated with the *Members*
+app.
+
+Smart buttons
+=============
+
+Towards the top of the :guilabel:`Contact` form there are some other options that are called *smart
+buttons* in Odoo.
+
+Here, Odoo displays records related to this contact that were created on other apps. Odoo integrates
+information from every single app, so there are many smart buttons.
+
+.. example::
+   For example, there is the :guilabel:`Opportunities` smart button, where all the opportunities
+   related to this customer will appear from the *CRM* app.
+
+.. tip::
+   If the corresponding applications are installed, their related smart buttons will appear
+   automatically.
+
+A user can see any :guilabel:`Meetings`, :guilabel:`Sales`, :guilabel:`POS Orders`,
+:guilabel:`Subscriptions`, project :guilabel:`Tasks`, and :guilabel:`More`. A user can even quickly
+access :guilabel:`Purchases`, :guilabel:`Helpdesk` tasks, :guilabel:`On-time Rate` for deliveries,
+:guilabel:`Invoiced` information, :guilabel:`Vendor Bills`, and the :guilabel:`Partner Ledger`
+connected to this contact.
+
+Deliveries, documents, loyalty cards, and direct debits will *also* link to smart buttons, like
+this, should there be any outstanding/on-file for this contact.
+
+There is also a :guilabel:`Go to Website` smart button. If the contact is a partner, the user will
+be able to visit their partner page on the Odoo-built website.
+
+Archive contacts
+----------------
+
+Now, if a user decides that they no longer want to have this contact active, the record can simply
+be archived. To do that, go to the :icon:`fa-cog` :guilabel:`Action` menu in the top-left, and click
+:guilabel:`Archive`.
+
+Then, click :guilabel:`Archive` again from the resulting :guilabel:`Confirmation` pop-up window.
+
+With this contact now successfully archived, as indicated by a banner at the top, they will not show
+up in the main contacts page, but they can still be searched for with the :guilabel:`Archived`
+filter.
+
+.. tip::
+   A contact can be unarchived if the user decides to work with them again. To do that, just click
+   the :icon:`fa-cog` :guilabel:`Action` menu again, and click :guilabel:`Unarchive`. As soon as
+   this is done, the :guilabel:`Archived` banner is removed, and the contact is back to normal.
+
+.. seealso::
+   Contact forms can be accessed in the *Sales* application, as well. Go to :menuselection:`Sales
+   app --> Orders --> Customers`, and clear any default filters from the search bar. Then, click on
+   the desired customer to open their contact form.
+
+   - :doc:`Add different addresses in CRM <../sales/sales/send_quotations/different_addresses>`
+   - `Odoo's eLearning Contacts tutorial
+     <https://www.odoo.com/slides/slide/contacts-2527?fullscreen=1>`_
+
+:icon:`fa-hand-spock-o`
```

Comment:
the only thing wrong with the icon is that it's not spinning :)

Reply:
100%